app/assets/sass/components/_figure.scss

```diff
@@ -1,3 +1,5 @@
+@use "node_modules/nhsuk-frontend/packages/nhsuk" as *;
```

Comment:
This’ll work, not sure if it’s better/best practice to import just the `core` styles, as it’s that’s all you need for a component.

Reply:
Yeah, I wasn't sure either. In another file we’re being even more specific and only `@use`ing `core/tools` and `core/settings`.

Also not sure what best practice is with namespacing...